### PR TITLE
SIGINT-1034: PR comment support for Polaris

### PR DESCRIPTION
**Pull request by spurohitsynopsys**

### CODE REVIEW

File: src/synopsys-action/tools-parameter.ts [hunk 1]
```diff
@@ -60,6 +60,27 @@ export class SynopsysToolsParameter {
       }
     }
 
+    if (parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED)) {
+      info('Polaris PR comment is enabled')
+      const prCommentSeverities: string[] = []
+      const inputPrCommentSeverities = inputs.POLARIS_PRCOMMENT_SEVERITIES
+      if (inputPrCommentSeverities != null && inputPrCommentSeverities.length > 0) {
```
Comment (kiransynopsys) [inline src/synopsys-action/tools-parameter.ts:67]: Can't be just to _if (inputPrCommentSeverities)_ from this  
_if (inputPrCommentSeverities != null && inputPrCommentSeverities.length > 0) _

File: src/synopsys-action/tools-parameter.ts [hunk 2]
```diff
@@ -60,6 +60,27 @@ export class SynopsysToolsParameter {
...
+        const severityValues = inputPrCommentSeverities.split(',')
+        for (const severity of severityValues) {
+          if (severity.trim().length > 0) {
```
Comment (kiransynopsys) [inline src/synopsys-action/tools-parameter.ts:70]: Can't we replace to  _if (severity.trim())_ to this.?

**Review by kiransynopsys — APPROVED** [review]

File: src/synopsys-action/tools-parameter.ts (hunk 1 above)
Comment (spurohitsynopsys) [reply src/synopsys-action/tools-parameter.ts:67]: want to be explicit and ensure that inputs.POLARIS_PRCOMMENT_SEVERITIES is both defined and contains a non-empty string

Comment (kiransynopsys) [reply src/synopsys-action/tools-parameter.ts:67]: we can use the truthy value, FYI ref : https://stackoverflow.com/a/5515349

Comment (spurohitsynopsys) [reply src/synopsys-action/tools-parameter.ts:67]: what if it contains only whitespaces

Comment (kiransynopsys) [reply src/synopsys-action/tools-parameter.ts:67]: If its working then okay,

File: src/synopsys-action/tools-parameter.ts (hunk 2 above)
Comment (spurohitsynopsys) [reply src/synopsys-action/tools-parameter.ts]: updated

**Review by lsynopsys — APPROVED** [review]

**Pull request merged**